mriqc/utils/misc.py

```diff
@@ -191,15 +187,15 @@ def generate_pred(derivatives_dir, output_dir, qctype):
     return out_csv
 
 
-def generate_csv(derivatives_dir, output_dir, qctype):
+def generate_csv(derivatives_dir, output_dir, mod):
     """
     Generates a csv file from all json files in the derivatives directory
     """
     errorlist = []
 
     # If some were found, generate the CSV file and group report
-    out_csv = op.join(output_dir, qctype[:4] + 'MRIQC.csv')
-    jsonfiles = glob(op.join(derivatives_dir, 'sub-*_%s.json' % QCTYPES[qctype[:4]]))
+    out_csv = op.join(output_dir, mod + 'MRIQC.csv')
```

Comment:
should we just remove the "MRIQC" suffix?, this way we would just have T1w.csv, T2w.csv and bold.csv files.